odxtools/singleecujob.py

```diff
@@ -167,11 +167,17 @@ def dop(self) -> Optional[DopBase]:
         return self._dop
 
 
+class ProgCodeSyntax(Enum):
```

Comment:
maybe use StrEnum?

Reply:
that seems to be not a part of the python standard library. While in this case, it would slightly better than `Enum`, adding a dependency is not worth it IMO...

Reply:
It is, but added in Python 3.11

https://docs.python.org/3.11/library/enum.html#enum.StrEnum